frappe/website/doctype/website_settings/website_settings.js

```diff
@@ -78,7 +77,8 @@ frappe.ui.form.on('Website Settings', {
 				main_items.push(d.label);
 			}
 		}
-		return main_items.join('\n');
+		frappe.meta.get_docfield("Top Bar Item", "parent_label", cur_frm.docname).options = main_items.join('\n');
```

Comment:
This is not the right place for the assignment to happen

Reply:
@scmmishra do you know the right place? pls indicate if so